llvm/utils/lit/lit/formats/googletest.py

```diff
@@ -43,7 +43,7 @@ def get_num_tests(self, path, litConfig, localConfig):
             return None
         return sum(
             map(
-                lambda line: lit.util.to_string(line).startswith("  "),
+                lambda line: line.decode("utf-8", errors="replace").startswith("  "),
```

Comment:
Can't we just use bytes here?
```suggestion
                lambda line: line.startswith(b"  "),
```